rest-tests/src/test/resources/features/file-transfer.feature

```diff
@@ -0,0 +1,72 @@
+@file-transfer
+Feature: File transfer
+
+  Background: Prepare
+    Given clean application state
+
+  @integrations-ftp-dropbox
+  Scenario: FTP to Dropbox
+    Given clean FTP server
+    Given deploy FTP server
+#    todo(sveres): this upload does not work, to be investigated where is the problem
```

Comment:
if it is not working, add "WIP" label to the PR

Reply:
PR works, since there is test.txt file on FTP server as default. this is just reminder.